SoC/gd32vf103/Board/gd32vf103v_rvstar/Include/gd32vf103v_rvstar.h

```diff
@@ -113,6 +116,16 @@ void gd_rvstar_key_init(key_typedef_enum keynum, keymode_typedef_enum keymode);
 uint8_t gd_rvstar_key_state_get(key_typedef_enum keynum);
 /* configure COM port */
 void gd_com_init(uint32_t usart_periph);
+
+/* Defines for LED functio/* Defines for LED / Key functions to new / general API */
+#define gd_led_init         gd_rvstar_led_init
```

Comment:
I prefer to change the old style API to new ones, and use macros to be back compatiable with previous.

In future, we can slowly deprecate the old APIs.